app/decorators/backend_api_decorator.rb

```diff
@@ -9,15 +9,31 @@ def api_selector_api_link
   end
 
   def add_backend_usage_backends_data
+    base_data.merge!({ updatedAt: updated_at })
+  end
+
+  def products_used_table_data
+    ServiceDecorator.decorate_collection(services.accessible.sort_by(&:name))
+                    .map(&:used_by_backend_table_data)
+                    .to_json
+  end
+
+  def base_data
     {
       id: id,
       name: name,
-      privateEndpoint: private_endpoint,
-      updatedAt: updated_at
+      privateEndpoint: private_endpoint
     }
   end
 
+  def products_used_table_data
+    ServiceDecorator.decorate_collection(services.accessible.sort_by(&:name))
+                    .map(&:used_by_backend_table_data)
+                    .to_json
+  end
+
   alias link api_selector_api_link
+  alias backends_used_table_data base_data
```

Comment:
why aliasing when you can rename the method directly?

Reply:
Because I want to make it clear that this is 2 things, the set of data that is common for all backend apis and also the data used for the "backends used table". Externally it's used like `backends_used_table_data`, internally `base_data`.